ms2/resources/schemas/dbscripts/postgresql/ms2-0.000-21.000.sql

```diff
@@ -416,6 +87,8 @@ ALTER TABLE ms2.Fractions ADD COLUMN MS2ScanCount INT;
 ALTER TABLE ms2.Fractions ADD COLUMN MS3ScanCount INT;
 ALTER TABLE ms2.Fractions ADD COLUMN MS4ScanCount INT;
 
+UPDATE ms2.fractions SET MzXmlUrl = 'file:///' || substr(MzXmlUrl, 7) WHERE MzXmlUrl LIKE 'file:/_%' AND MzXmlUrl NOT LIKE 'file:///%' AND MzXmlUrl IS NOT NULL;
```

Comment:
Feel free to delete this line - the table will always be empty when bootstrapping.
```suggestion
```